src/components/AuthModal.vue

```diff
@@ -20,17 +20,19 @@
         </ion-grid>
         <ion-item>
           <ion-label position="floating">Username</ion-label>
-          <ion-input type="text" v-model="username"></ion-input>
+          <ion-input type="text" required v-model="username"></ion-input>
         </ion-item>
         <ion-item>
           <ion-label position="floating">Password</ion-label>
-          <ion-input type="password" v-model="password"></ion-input>
+          <ion-input type="password" required v-model="password"></ion-input>
         </ion-item>
         <ion-grid>
           <ion-row>
             <ion-col style="text-align:center;">
-              <ion-button size="medium" expand="block" @click="logIn">
-                <ion-icon :icon="lockClosed" style="font-size:15px;padding-right:10px;"></ion-icon>Sign In
+              <ion-button size="medium" expand="block" :disabled="!username || !password" @click="logIn">
+                <ion-icon v-if="!loading" :icon="lockClosed" style="font-size:15px;padding-right:10px;"></ion-icon>
+                <ion-spinner v-if="loading" style="margin-right:10px"></ion-spinner>
```

Comment:
Looks like we need this line to be changed to shrink the spinner - maybe an `em` measurement for the height and width might work well?

Reply:
@locness3

Reply:
I'll try `em`

Reply:
![image](https://user-images.githubusercontent.com/37651007/107667387-d1263f80-6c8f-11eb-8ccb-bce73450871d.png)
Like this? @micahlt